File: Easy/1D Bush Fire/1D Bush Fire.cs
using System;

class Solution
{
    static void Main(string[] args)
    {
        int N = int.Parse(Console.ReadLine());
        
        for (int i = 0; i < N; i++)
        {
            int answer = 0;
            char[] line = Console.ReadLine().ToCharArray();
            for (int j = 0; j < line.Length; j++)
            {
                if (line[j] == 'f')
                {
                    answer+=1;
                    j+=2;
                }
            }
            Console.WriteLine(answer);
        }
    }
}

File: Easy/1D Bush Fire/1D Bush Fire.py
n = int(input())
for i in range(n):
    answer = 0
    u = 0
    line = input()
    while u < len(line):
        if line[u] == 'f':
            answer+=1
            u+=2
        u+=1
    print(answer)
exit(0)
